crypto/keys/secp256k1/internal/secp256k1/curve.go

```diff
@@ -143,34 +143,34 @@ func (bitCurve *BitCurve) Add(x1, y1, x2, y2 *big.Int) (*big.Int, *big.Int) {
 
 // addJacobian takes two points in Jacobian coordinates, (x1, y1, z1) and
 // (x2, y2, z2) and returns their sum, also in Jacobian form.
-func (BitCurve *BitCurve) addJacobian(x1, y1, z1, x2, y2, z2 *big.Int) (*big.Int, *big.Int, *big.Int) {
```

Comment:
can we no lint this file. its a fork so shouldn't need to touch this code

Reply:
I am going to diff this folder against main to make sure no further issues exist